addon/components/validation-error-field.js

```diff
@@ -6,12 +6,18 @@ var ValidationErrorField = Ember.Component.extend({
     attributeBindings: ["array", "className", "model", "field", "validation", "submitted", "delayed", "index"],
     initialize: Ember.on("init", function() {
         var field = this.get("field");
-        Ember.Binding.from("model." + field).to("fieldName").connect(this);
-        Ember.Binding.from("model." + field + "IsPrimed").to("isPrimed").connect(this);
-
         var computedKeys = ["validation", "isPrimed", "submitted", "fieldName"];
+
         if(this.get("array")) {
-            computedKeys.push("array.@each." + field);
+            var fields = field.split(",");
+            fields.forEach(function(f) {
+                computedKeys.push("array.@each." + f);
+            });
+            Ember.Binding.from("model." + fields[0] + "IsPrimed").to("isPrimed").connect(this);
+            Ember.Binding.from("model." + fields[0]).to("fieldName").connect(this);
```

Comment:
I updated this in the latest commit - isPrimed was still required but I was able to remove fieldName from the array supported scenario as it's not used inside the components visible function :) less code to maintain/read/etc so thanks for the call out!